templates/jmxtrans.service.epp

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=JMX Transformer - more than meets the eye
+After=syslog.target network.target
+
+[Service]
+Type=forking
+User=<%= $user %>
+# Run ExecStartPre with root-permissions
+PermissionsStartOnly=true
+ExecStartPre=-/usr/bin/mkdir /run/jmxtrans/
+ExecStartPre=/usr/bin/chown -R <%= $user %> /run/jmxtrans/
+PIDFile=/var/run/jmxtrans/jmxtrans.pid
+ExecStart=/usr/share/jmxtrans/bin/jmxtrans start
```

Comment:
So, should this be platform dependant then? `/usr/share/jmxtrans/bin/jmxtrans.sh start` on Debian, `/usr/share/jmxtrans/bin/jmxtrans start` on RedHat?